contracts/pools/AAVEDAIDonorsPool.sol

```diff
@@ -14,70 +14,60 @@ import "../tokens/IToken.sol";
 import "../aragon/ITokenManager.sol";
 import "./IDonor.sol";
 
-
 // this is a basic AAVE donors pool for DAI
 // @todo add more documentation
 contract AAVEDAIDonorsPool is Ownable, IDonor {
-
     using SafeMath for uint256;
-    
+
+    // @todo add more documentation
     // hardcoding dai for hackathon simplicity
     IERC20 public dai = IERC20(0x001B3B4d0F3714Ca98ba10F6042DaEbF0B1B7b6F);
     IAToken public aToken = IAToken(0x639cB7b21ee2161DF9c882483C9D55c90c20Ca3e);
     ILendingPool public aaveLendingPool = ILendingPool(0x9198F13B08E299d85E096929fA9781A1E3d5d827);
     ITokenManager private tokenManager = ITokenManager(0xA6687fe9F472159be142b584D3756634C2A1883a);
 
     IToken public token;
-    
+
     mapping(address => uint256) public deposits;
     mapping(address => uint256) public donationPercentages;
-    mapping(uint256=>bool) public donated;
+    mapping(uint256 => bool) public override donated;
 
```

Comment:
remove override

Reply:
Notwithstanding the linter generates this error ```Overriding public state variable is missing "override" specifier.``` ? lets see if it compiles

Reply:
ok no worries